demo/app.py

```diff
@@ -0,0 +1,94 @@
+import adddeps
+
+import streamlit as st
+import cv2
+import io
+import re
+import numpy as np
+import torch
+import torchvision.transforms as T
+
+from src.dataset import CLASSES
+from src.models import get_multilabel_model
+from src.utils import inference
+
+def load_model(model_file, model_path):
+    out = re.findall("[a-z]*net[0-9]*", model_path)
+    if len(out) == 0:
+        model_name = 'resnet18'
+    else:
+        model_name = out[0]
+    num_classes = len(CLASSES)
+    model = get_multilabel_model(model_name, num_classes=num_classes, weights=None)
+
+    model.load_state_dict(torch.load(model_file, map_location='cpu'))
+    model.eval()
+    return model
+
+class App(object):
+    def __init__(self):
+        self.conf_threshold = 0.5
+        self.model = None
+        self.classes = []
+        self.input_height = 512
+        self.input_width = 512
```

Comment:
we train model on 256x256, lets do the inference on it as well

Reply:
ok, will fix it